### PR TITLE
➕ Add husky and lint-staged

### PR DESCRIPTION
**Pull request by JuKahl**
- Installed husky with `npm install husky --save-dev && npx husky init husky`
- Installed lint-staged with `npm i -D lint-staged lint-staged`
- Added `.lintstagedrc.json`
- Edited `.husky/pre-commit`
- Added scripts `prepare`,`test` &` typecheck`
- Added `pre-push` with `npx husky add .husky/pre-push "npm test"`

> You can skip verification with --no-verify in your git commands.

https://typicode.github.io/husky/
https://github.com/okonet/lint-staged

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -9,7 +9,10 @@
     "eslint": "eslint .",
     "prettify": "prettier --write .",
     "stylelint": "stylelint \"**/*.css\"",
-    "stylelint-check": "stylelint-config-prettier-check"
+    "stylelint-check": "stylelint-config-prettier-check",
+    "prepare": "husky install",
+    "typecheck": "tsc",
+    "test": "npm run eslint && npm run stylelint && npm run prettify && npm run typecheck"
```
Comment (fre-ben) [inline package.json]: `npm run prettify` runs `--write` isntead of `--check`.
Review line 10, your prettify script.

**Review by fre-ben — CHANGES_REQUESTED** [review]

**Review by christophvogel — APPROVED** [review]
LGTM 👍

**Comment by DNaunin** [comment]
Good Job! Only one thing: 
You did not add a prettify check in the package.json, so your husky test will make the prettify changes directly instead of only checking them, maybe this is deliberate.

File: package.json (hunk 1 above)
Comment (JuKahl) [reply package.json]: Thank you!

**Review by fre-ben — APPROVED** [review]
Another amazing job done 💯

**Review by cboenke — APPROVED** [review]
LGTM 🚀

**Pull request merged**